src/textual/widgets/_text_log.py

```diff
@@ -27,7 +30,7 @@ class TextLog(ScrollView, can_focus=True):
     }
     """
 
-    max_lines: var[int | None] = var(None)
+    max_lines: var[int | None] = var[Optional[int]](None)
```

Comment:
As far as my understanding goes, `var` is just an "alias" for a quick way to create a reactive attribute with some parameters in pre-defined values.
Would it be worth typing `max_lines` as `max_lines: reactive[int | None] = var[Optional[int]](None)`?
(This was not a rhetoric question.)

Reply:
What benefit do we get from that? A downside I can see is that it might be confusing to a reader of the code, seeing two "different" types like that.